### PR TITLE
Support OTA ImagePageRequest, 

### PR DESCRIPTION
**Pull request by felixstorm**
* updates OTA implementation to support ImagePageRequest (in addition to ImageBlockRequest)
* enables OTA for Insta/Jung wall switch (see https://github.com/Koenkk/zigbee-OTA/commit/0350533133f48aaf030cda113287fe63e7e08749) - was a bit more work than expected since it wouldn't update without ImagePageRequest
* change manufacturer from 'Gira' to 'Insta/Gira/Jung' (Insta is the OEM for Gira & Jung)

Follow-up for PR https://github.com/Koenkk/zigbee-herdsman/pull/156

As always: feel free to change as you like and thanks a lot for you work!

### CODE REVIEW

File: ota/common.js [hunk 1]
```diff
@@ -107,10 +108,15 @@ async function requestOTA(endpoint) {
     }
 }
 
-function getImageBlockResponsePayload(image, imageBlockRequest) {
-    const start = imageBlockRequest.payload.fileOffset;
+function getImageBlockResponsePayload(image, imageBlockRequest, pageOffset, pageSize) {
+    const start = imageBlockRequest.payload.fileOffset + pageOffset;
     // When the data size is too big, OTA gets unstable.
-    let end = start + Math.min(50, imageBlockRequest.payload.maximumDataSize);
```
Comment (Koenkk) [inline ota/common.js]: I think we should only limit the datasize to 40 for insta devices (based on the manufacturerCode), lower data size = longer update time. Now this affects all devices.

File: devices.js [hunk 2]
```diff
@@ -6877,18 +6877,19 @@ const devices = [
         },
     },
 
-    // Gira
+    // Insta/Gira/Jung
     {
         zigbeeModel: [' Remote'],
         model: '2430-100',
-        vendor: 'Gira',
+        vendor: 'Insta/Gira/Jung',
```
Comment (Koenkk) [inline devices.js]: Are the Gira and Jung white-labelled devices? if so we can mark them as such
- example: https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/devices.js#L1044
- See https://www.zigbee2mqtt.io/information/supported_devices.html#zemismart ZM-CSW002-D how it shows up in the docs.

File: ota/common.js (hunk 1 above)
Comment (felixstorm) [reply ota/common.js]: No problem - is the updated code in https://github.com/Koenkk/zigbee-herdsman-converters/pull/1082/commits/6f07dc2fa19eaad6c1b574e2958c52f76a3290f5 ok for you?

File: devices.js (hunk 2 above)
Comment (felixstorm) [reply devices.js]: Yes, they are both white-labelled devices originally manufactured by [Insta](https://www.insta.de/instastorefront/en/). If you're ok with https://github.com/Koenkk/zigbee-herdsman-converters/pull/1082/commits/b74264fd32316b713cc7274dfb1ebe8278a526c3, then I'll file another PR to change the device model in https://github.com/Koenkk/zigbee2mqtt/blob/master/lib/extension/homeassistant.js#L944 also.

**Comment by Koenkk** [comment]
Thanks again! :smile:

**Pull request merged**